testing/sdk_utils.py

```diff
@@ -109,6 +112,20 @@ def is_strict_mode():
     return os.environ.get('SECURITY', '') == 'strict'
 
 
+def random_string(length=8):
+    return ''.join(
```

Comment:
Why not just use `str(uuid.uuid4())`?

Reply:
This is easier if debugging for service uid / secret paths. Also wherever needed, we can just use `uuid.uuid()` directly instead of calling `random_string()`.